src/views/LimitedRestorationExtension.vue

```diff
@@ -337,7 +338,7 @@ export default class LimitedRestorationExtension extends Vue {
 
   get applicantEmail (): string {
     const currentApplicant = this.getOrgPeople.filter(orgPerson => !this.wasRemoved(orgPerson))
-    return currentApplicant[0].officer.email
+    return currentApplicant[0]?.officer.email
```

Comment:
👍

Reply:
I've left this one, if you want it reverted let me know

Reply:
I like this one :)